bloom-housing/bloom
# 2056/user account edit fix

# Pull Request Template

## Issue Overview

This PR addresses #2056


## Description

Updates settings form to use a new user service. It fixes problems with changing email.

## How Can This Be Tested/Reviewed?

1. Create or login into the account
2. Try to change name, dob, password
3. Make sure that details have been saved correctly
4. Try to change an email
5. Make sure that you've received a confirmation email
6. Check if after confirmation you are able to log in using a new email

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
